components/k8s-model-server/istio-integration.md

```diff
@@ -0,0 +1,42 @@
+# Istio integration for TF serving
+
+[Istio](https://istio.io/) provides a lot of functionalities that we want to have, such as metrics, auth and
+quota, rollout and A/B testing. We have an [issue](https://github.com/kubeflow/kubeflow/issues/464) to track
+the progress.
+
+## Install Istio
+Following the istio [doc](https://istio.io/docs/setup/kubernetes/quick-start.html#installation-steps).
+After the installation, you should see services istio-pilot and istio-mixer in namespace istio-system.
+
+## Kubeflow TF Serving with Istio
+
+This document is working in progress.
+
+Istio by default [denies egress traffic](https://istio.io/docs/tasks/traffic-management/egress.html).
```

Comment:
Does egress traffic from TFServing get routed through ISTIO in addition to ingress traffic? How does this happen?

Reply:
I guess the init container takes over the iptable. Does it make sense?